api/src/troubleshoot/collector.ts

```diff
@@ -9,8 +9,10 @@ export class Collector {
 export async function injectKotsCollectors(parsedSpec: any, licenseData: string): Promise<any> {
   let spec = parsedSpec;
   spec = await injectDBCollector(spec);
-  spec = await injectLicenseCollector(spec, licenseData);
-  spec = await injectAPICollector(spec);
+  spec = injectLicenseCollector(spec, licenseData);
+  spec = injectAPICollector(spec);
+  spec = injectOperatorCollector(spec);
+  spec = injectRookCollectors(spec);
```

Comment:
@emosbaugh if we make this conditionally injected, only when the ENABLE_KURL variable is set, then we won't see errors in the support bundles for existing cluster installs. While the errors are generally harmless, I think only including this collector for installs with ENABLE_KURL set will be beneficial in the troubleshooting process. It will help differentiate between errors.json meaning "rook is broken or rbac can't access" vs "there is no rook and should be" vs "there is no rook but that's expected".  This last case of "error, not really an error" is what i'm trying to avoid.

Reply:
there are no errors. its probably not working like you expect. ill exclude regardless

Reply:
thanks @emosbaugh !